ts/Extensions/OfflineExporting/OfflineExporting.ts

```diff
@@ -889,6 +904,14 @@ namespace OfflineExporting {
                 [width, height]
             );
 
+        // Apply new font if the fontFamily set, #6417.
+        if (options.chartOptions &&
+            options.chartOptions.chart.style &&
+            options.chartOptions.chart.style.fontFamily
+        ) {
+            pdf.setFont(options.chartOptions.chart.style.fontFamily);
+        }
+
```

Comment:
I'm a bit in doubt here... What happens if the implementer sets `chart.style.fontFamily` to something else, like `Arial, verdana, sans-serif`? It may be that we need to separate the PDF-specific font family from they style option. In that case we should combine it with the URL:

```
exporting: {
  pdfFont: {
    name: 'NotoSans-Regular',
    url: 'fontfile.js'
  }
}
```

In fact, I think that looks much cleaner and also has the benefit that all the configuration is done in one place.